bikeshed/unsortedJunk.py

```diff
@@ -1207,12 +1207,17 @@ def cleanupHTML(doc: "t.SpecType"):
             el.tag = "span"
             h.addClass(el, "css")
 
+        # Assert IDs are used to tag arbitrary sections with an ID so you can point tests at it.
+        # (The ID will be guaranteed stable across publications, but guaranteed to change when the text changes.)
+        #
         # Transform the <assert> fake tag into a span with a unique ID based on its contents.
-        # This is just used to tag arbitrary sections with an ID so you can point tests at it.
-        # (And the ID will be guaranteed stable across publications, but guaranteed to change when the text changes.)
         if el.tag == "assert":
             el.tag = "span"
             el.set("id", h.safeID(doc, "assert-" + h.hashContents(el)))
+        # For any <div assert> elements, add a unique ID based on its contents.
+        if el.tag == "div" and "assert" in el.attrib:
```

Comment:
Use `el.get("assert") is not None` here, to match style elsewhere.

Reply:
done